package.json

```
@@ -12,7 +12,7 @@
   "dependencies": {
     "@across-protocol/boba-sdk": "^1.0.4",
     "@across-protocol/contracts-v2": "^2.0.2",
-    "@across-protocol/sdk-v2": "^0.4.1",
+    "@across-protocol/sdk-v2": "^0.4.2",
```

```suggestion
    "@across-protocol/sdk-v2": "^0.4.3",
```

In order to incorporate the UBA types added in this commit: https://github.com/across-protocol/sdk-v2/commit/f4e6c4d429d3bf2c521b3b5d7a6def7e684c6a95.